app/views/course/achievements/show.html.slim

```diff
@@ -0,0 +1,15 @@
+- add_breadcrumb format_inline_text(@achievement.title)
+= page_header format_inline_text(@achievement.title)
+
+= render partial: 'achievements',
+         locals: { include_requirements: false, achievements: [@achievement] }
+
+div.user-achievements
+  h2 = t('.secondary_header')
+  div.row
+    - @achievement.course_users.includes(:user).each do |course_user|
+      div.col-xs-4.col-sm-3.col-md-2.col-lg-1
+        = div_for(course_user, class: ['text-center']) do
```

Comment:
You should not text center here, make a sass class and use that here instead so other people can style